shared/src/main/scala/scala/util/parsing/combinator/Parsers.scala

```diff
@@ -663,8 +663,9 @@ trait Parsers {
    *  @param  es the list of expected elements
    *  @return a Parser that recognizes a specified list of elements
    */
-  def acceptSeq[ES](es: ES)(implicit f: ES => Iterable[Elem]): Parser[List[Elem]] =
-    es.foldRight[Parser[List[Elem]]](success(Nil)){(x, pxs) => accept(x) ~ pxs ^^ mkList}
+  def acceptSeq[ES](es: ES)(implicit f: ES => Iterable[Elem]): Parser[List[Elem]] = {
+    f(es).foldRight[Parser[List[Elem]]](success(Nil)){(x, pxs) => accept(x) ~ pxs ^^ mkList}
```

Comment:
this is also interesting, might be worth checking with @smarter if expected

Reply:
Indeed, all three errors and the workarounds needed may be worth a look I think.

Reply:
it could be because the implicit is merely a `Function1` and not the new `Conversion` type, but I would have thought `-language:Scala2` would allow it. in any case, I think it merits a comment